content/sensu-go/5.18/release-notes.md

```diff
@@ -53,6 +54,28 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.18.0 release notes
+
+**February 24, 2020** &mdash; The latest release of Sensu Go, version 5.18.0, is now available for download.
+**PLACEHOLDER FOR RELEASE SUMMARY**
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.18.0.
+
+**IMPROVEMENTS:**
+
+- The `event.entity.entity_class` value now defaults to `proxy` for [`POST /events`][114] requests.
+- Proxy entities are now automatically created when events are published with an entity that does not exist.
+- Supported Bonsai asset versions are now prefixed with the letter `v` (for example, `v1.2.0`).
```

Comment:
```suggestion
- Sensuctl now accepts supported Bonsai asset versions, which are prefixed with the letter `v` (for example, `v1.2.0`).
```

Reply:
What do you mean by `supported Bonsai asset versions`?

Reply:
I think I was misunderstanding the changelog item (Support Bonsai assets versions prefixed with the letter v) -- I thought support meant the v designated a specific supported asset rather than "Sensu now supports..." I get it now, thank you. I will fix it.